test/foundry/deploy/15_holograph_registry_tests.t.sol

```diff
@@ -154,15 +153,15 @@ contract HolographRegistryTests is Test {
    * Refers to the hardhat test with the description 'Should return valid contracts'
    */
   function testReturnValidHolographableContract() public view {
-    // TODO Change the address of sampleErc721Holographer to the address obtained from the hash
     address[] memory contracts = registryDeployedByScript.getHolographableContracts(
       0,
       expectedHolographableContractsCount
     );
     assertEq(contracts.length, expectedHolographableContractsCount);
+    address add = address(registryDeployedByScript.getHolographedHashAddress(erc721ConfigHash));
```

Comment:
which means add? maybe Erc721Holographer? or Erc721HolographerAddress?

Reply:
It is the address obtained from the hash. I renamed it to `hashAddress`